tests/backends/mysql/tests.py

```diff
@@ -89,3 +89,19 @@ def test_isolation_level_validation(self):
         )
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
             new_connection.cursor()
+
+
+@unittest.skipUnless(connection.vendor == 'mysql', 'MySQL tests')
```

Comment:
I'd put this in a separate `test_schema.py` file -- but is it really MySQL-specific? I guess some database might quote those values differently but I didn't check. I would put the non-regression tests (everything except the bytestring?) in a separate commit that precedes the bug fix so it's more clear exactly which case this fixes. `return self.connection.connection.escape(value, self.connection.connection.encoders)` might be considered as a separate refactoring commit also.